README.md

```diff
@@ -1,79 +1,86 @@
-
 ## License
 
 The *INTX Python SDK* sample library is free and open source and released under the [Apache License, Version 2.0](LICENSE).
 
 The application and code are only available for demonstration purposes.
 
-## Usage
+## Installation
 
-### Setting Up Credentials
+```bash
+pip install intx-sdk-py
+```
 
-To use the *INTX Python SDK*, initialize the [Credentials](intx_sdk/credentials.py) class with your INTX API credentials. This class is designed to facilitate the secure handling of sensitive information required to authenticate API requests.
+Or install from source:
 
-Ensure that your API credentials are stored securely and are not hard-coded directly in your source code. The Credentials class supports creating credentials from a JSON string or directly from environment variables, providing flexibility and enhancing security.
+```bash
+git clone https://github.com/coinbase-samples/intx-sdk-py.git
+cd intx-sdk-py
+pip install -e .
+```
 
-#### Example Initialization:
+## Authentication
 
-```python
-from intx_sdk.credentials import Credentials
+To use the INTX Python SDK, you will need to create API credentials in the [INTX web console](https://international.coinbase.com/) under Settings -> API.
 
-credentials = Credentials.from_env("INTX_CREDENTIALS")
+Credentials can be stored as environment variables or passed directly. The SDK supports two initialization patterns:
+
+### Using .env.example (Recommended)
+
+Copy the example file to `.env` and then fill in your credentials:
+
+```bash
+cp .env.example .env
+# then edit .env and set your values
 ```
 
-#### Specifying the base url
-INTX supports both production and sandbox environments. You can specify the desired environment by setting the base URL when initializing the `IntxClient`:
+Initialize the client:
+
 ```python
-intx_client = IntxClient(credentials, base_url="https://api-n5e1.coinbase.com/api/v1")
+from intx_sdk import IntxServicesClient
+
+client = IntxServicesClient.from_env()
 ```
-If no `base_url` is provided, the default production URL will be used.
 
-#### Environment Variable Format: 
+### Using Credentials Object
 
-The JSON format expected for `INTX_CREDENTIALS` is:
+```python
+from intx_sdk import IntxServicesClient
+from intx_sdk.credentials import Credentials
 
-```
-{
-  "accessKey": "",
-  "passphrase": "",
-  "signingKey": "",
-}
+credentials = Credentials(
+    access_key="your-access-key",
+    passphrase="your-passphrase",
+    signing_key="your-signing-key"
+)
+
+client = IntxServicesClient(credentials)
 ```
 
-### Obtaining API Credentials 
+## Environment Configuration
 
-Coinbase INTX API credentials can be created in the INTX web console under Settings -> APIs. 
+By default, the SDK uses the production environment. To use the sandbox environment, set the `INTX_ENVIRONMENT` environment variable:
```

Comment:
Do we still have the ability to override the URL? If not, we should support like the other SDKs